src/gui/attributetable/qgsdualview.cpp

```diff
@@ -943,13 +954,45 @@ void QgsDualView::resizeColumn()
   }
 }
 
+void QgsDualView::resizeAllColumns()
+{
+  QAction *action = qobject_cast<QAction *>( sender() );
+  int col = action->data().toInt();
+  if ( col < 0 )
+    return;
+
```

Comment:
```suggestion
```

Reply:
I guess this is related to the previous comment. If I was able to convince you on my previous point about retaining the setData() line, then I think these lines should stay as well?

Reply:
correct